docs/included.md

```diff
@@ -215,6 +215,14 @@
     [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
   - 
     [Accuracy (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#ACC)
+* -  
+    - Base rate
```

Comment:
For consistency with the rest of the table, should this be "Base Rate" with a capital "R"?